solr/core/src/java/org/apache/solr/security/PKIAuthenticationPlugin.java

```diff
@@ -347,22 +348,27 @@ PublicKey fetchPublicKeyFromRemote(String nodename) {
     HttpEntity entity = null;
     try {
       String uri = url + PublicKeyHandler.PATH + "?wt=json&omitHeader=true";
+      ModifiableSolrParams solrParams = new ModifiableSolrParams();
```

Comment:
glad to see you convert to use a SolrClient :-).  It'll be nice to have the observability benefits.